srcs/juloo.keyboard2/KeyValue.java

```diff
@@ -550,7 +550,7 @@ public static KeyValue getKeyByName(String name)
       case "meteg_placeholder": return placeholderKey(Placeholder.METEG);
       /* intending/preventing ligature - supported by many scripts*/
       case "zwj": return charKey("zwj", '\u200D', 0); // zero-width joiner (provides ligature)
-      case "zwnj": return charKey("zwnj", '\u200C', 0); // zero-width non joiner (prevents unintended ligature)
+      case "halfspace": return charKey("⸽", '\u200C', 0);
```

Comment:
We still need to support layouts that were using `zwnj` until now:

```suggestion
      case "zwnj":
      case "halfspace": return charKey("⸽", '\u200C', 0); // zero-width non joiner
```